src/index.js

```diff
@@ -103,9 +104,15 @@ const wrappedFetch = async (url, options = DEFAULT_FETCH_OPTIONS) => {
     }
   }
 
+  if (opts.qs) {
+    urlWithQuery = new URL(url);
+    Object.keys(opts.qs).forEach((key) => urlWithQuery.searchParams.append(key, opts.qs[key]));
```

Comment:
use `Object.entries`

Reply:
maybe also support arrays? eg:
```
qs: {
  name: ['foo', 'bar']
}
```

Reply:
I'll make another feature request for the array querystring :) @tripodsan